src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java

```diff
@@ -332,6 +333,73 @@ public void responseValidateV2(Map<String, Object> orgResponse) {
 
     }
 
+    public void validateAccessTypesInRetrievedUser(Map<String, Object> searchResponse, String expectedStatus,
+                                                   Boolean rolesReturned) {
+        assertThat(searchResponse.get("users")).asList().isNotEmpty();
+        assertThat(searchResponse.get("organisationIdentifier")).isNotNull();
+        assertThat(searchResponse.get("organisationProfileIds")).isNotNull();
+        List<HashMap> professionalUsersResponses = (List<HashMap>) searchResponse.get("users");
+        List<HashMap> userAccessTypeList = (List<HashMap>) professionalUsersResponses.get(0).get("userAccessTypes");
+
+        assertThat(userAccessTypeList).isNotEmpty();
+        assertThat(userAccessTypeList).hasSize(1);
+
+        professionalUsersResponses.forEach(user -> {
```

Comment:
since size of professionalUsersResponses is 1, get map for index 0 and assert for single object ( remove forEach)

Reply:
Thank you for review. Its fixed now. please review.